script/input/1/common.json

```diff
@@ -0,0 +1,6 @@
+{
+    "ilk": "ALLOCATOR-SPARK-1",
+    "usdcUnitSize": 100,
+    "usdsUnitSize": 100,
```

Comment:
For mainnet we will be using real tokens, so let's set this to $10 so that the amount filled into the PSM is $100.

Reply:
Updated